lib/assets/javascripts/views/devise/devise.js

```diff
@@ -0,0 +1,40 @@
+import ariatiseForm from '../../utils/ariatiseForm';
+import { addMatchingPasswordValidator, addShowHidePasswordControl } from '../../utils/passwordHelper';
+import { SHOW_SELECT_ORG_MESSAGE, SHOW_OTHER_ORG_MESSAGE } as constants from '../constants';
+
+/*
+  Ariatise all of the devise forms and attach a
+  @param value to check
+  @return true or false
+*/
+if (document) {
+  document.addEventListener('DOMContentLoaded', () => {
```

Comment:
this is vanilla JS, we are using jQuery library for adding listeners, we have to be consistent, either vanilla JS or Jquery but not both, please. I would be more inclined to use vanilla JS since browsers are more consistent nowadays, we can discuss this in our weekly dev's meeting.

Reply:
ok. I was having trouble with this. It seems as though jQuery isn't ready at the point this code was executing.